app/models/health/release_form.rb

```diff
@@ -29,6 +29,7 @@ class ReleaseForm < HealthBase
     include HealthFiles
 
     validates :signature_on, presence: true
+    validates :participation_signature_on, presence: true
```

Comment:
so both signatures are required, always?  I guess that might clear up some of my previous notes.